File: sanidad-backend/src/main/java/com/farmacia/sanidadbackend/dto/DevolucionProveedorDetalleResponse.java
package com.farmacia.sanidadbackend.dto;

import lombok.Data;

@Data
public class DevolucionProveedorDetalleResponse {

    private Long id;
    private Long loteDetalleId;
    private String medicamentoNombre;
    private String loteNumero;
    private Integer cantidadDevuelta;
}

File: SanidadMovil/app/src/main/java/com/sanidad/movil/data/remote/dto/DevolucionProveedorDetalleResponse.kt
package com.sanidad.movil.data.remote.dto

data class DevolucionProveedorDetalleResponse(
    val id: Long,
    val loteDetalleId: Long,
    val medicamentoNombre: String,
    val loteNumero: String,
    val cantidadDevuelta: Int
)